skyrl-train/skyrl_train/inference_engines/sglang/ipc_utils.py

```diff
@@ -0,0 +1,55 @@
+"""Utilities for IPC request serialization for SGLang weight transfer."""
```

Comment:
the reason why these functions are extracted in a separate file is to enable running tests without GPUs.
`import sglang_engine` now will fail without GPUs because of GPU-only dependencies. 
Ideally, we should avoid importing GPU-only deps at the module level.